dali/ft/filecopy.cpp

```diff
@@ -2623,6 +2623,49 @@ void FileSprayer::setTarget(IDistributedFile * target)
     }
 }
 
+void FileSprayer::checkTargetPath(RemoteFilename & filename)
+{
+    StringBuffer targetFilePath;
+
+    filename.getRemotePath(targetFilePath);
+    // prune the leading "//" from remote path
+    targetFilePath.remove(0, 2);
+
+    StringBuffer netaddress;
+    filename.queryIP().getIpText(netaddress);
+    const char * pnetaddress = netaddress.str();
+
+    if (streq("127.0.0.1", pnetaddress))
```

Comment:
The loopback ip can be different.
I think there's an existing call to check if IP is the loopback in jsocket.